hello-world/package.json

```diff
@@ -4,13 +4,15 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "archive": "cd .. && bun pm pack && cd hello-world && mkdir -p lib && mv ../*.tgz ./lib/latest.tgz",
+    "install-arch": "bun add ./lib/latest.tgz -d --trust",
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview"
   },
   "devDependencies": {
     "vite": "^6.2.0",
-    "vite-plugin-fable": "file:..",
+    "vite-plugin-fable": "./lib/latest.tgz",
```

Comment:
tgz seems weird, can't you use `../`?

Reply:
i tried but i had issues with recursion as it seems to install the whole univers in node_modules/vite-plugin-fable/...../node_modules/vite-plugin-fable.... and adding a bunch fo files even though they are not mentioned in `files` in root `package.json`, so i found this solution to be a bit better/slimmer and not having that recursive issue

Reply:
I see, yeah that is probably why I went with `import plugin from "../index.js"` instead.